Enterprise-CMCS/macpro-mdct-mfp
# Mdct 2959 - real statusing on entity rows

### Description
Update transition benchmarks dashboard to include real statuses [(figma)](https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=0%3A1&mode=design&t=PshqgynVZUuwPlcI-1) 

status will be defined on the front end if all data is entered for that target population

Also fixed some styling around the color of borders from light to lighter gray and cleaned up props and styling brought over from MCR that we aren't using in MFP.

### Related ticket(s)


MDCT-2959

---
### How to test
go to `Transition Benchmarks`, see that all the target populations are requiring completion. Once you enter a drawer and `Save & Close`, see that the entity is now displaying a complete status. Same goes for creating a new target population.


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
